subiquity/server/controllers/ubuntu_advantage.py

```diff
@@ -15,9 +15,23 @@
 """ Module defining the server-side controller class for Ubuntu Advantage. """
 
 import logging
+import os
 
 from subiquity.common.apidef import API
-from subiquity.common.types import UbuntuAdvantageInfo
+from subiquity.common.types import (
+    UbuntuAdvantageInfo,
+    UbuntuAdvantageCheckTokenAnswer,
+    UbuntuAdvantageCheckTokenStatus,
+)
+from subiquity.common.ubuntu_advantage import (
```

Comment:
Pedantry, but this module can move to subiquity.server now I think?

Reply:
True!
Maybe it would make sense to keep the exception types (e.g., `InvalidUAToeknError`, `ExpiredUATokenError`, `CheckSubscriptionError`) under `common` so they can be reused on the client side (which we're not doing ATM) but we can come back to it later and move them to common if needed. I'll move everything under `server` for now.